src/test/ui/consts/const_in_pattern/cross-crate-fail.rs

```diff
@@ -0,0 +1,16 @@
+// aux-build:consts.rs
+
+#![warn(indirect_structural_match)]
+
+extern crate consts;
+use consts::*;
+
+fn main() {
+    match None {
+        SOME => panic!(),
```

Comment:
maybe write the pattern as `consts::SOME` to make it crystal clear to future readers that this is not a binding to a local ident?